srcpkgs/guile3/template

```diff
@@ -0,0 +1,49 @@
+# Template file for 'guile3'
+pkgname=guile3
+version=3.0.4
+revision=1
+wrksrc="guile-${version}"
+build_style=gnu-configure
+configure_args="--program-suffix=3"
+hostmakedepends="pkg-config texinfo"
+makedepends="gc-devel gmp-devel readline-devel libatomic_ops-devel
+ libffi-devel libltdl-devel libunistring-devel"
+short_desc="Portable, embeddable Scheme implementation written in C"
+maintainer="Evgeny Ermakov <evgeny.v.ermakov@gmail.com>"
+license="GPL-3.0-or-later, LGPL-3.0-or-later"
+homepage="https://www.gnu.org/software/guile"
+distfiles="${GNU_SITE}/guile/guile-${version}.tar.gz"
+checksum=13f2f61a665469d330e651c8e9f7faecbb87fe474767f9532f2e821701730957
+conflicts="guile" # /usr/share/info/{guile.info*,r5rs.info}
+
+if [ "$CROSS_BUILD" ]; then
+	# Need host guile to cross compile
+	hostmakedepends+=" guile3"
+	export GUILE_FOR_BUILD=guile3
+fi
+
+do_check() {
+	# TODO: fix failing tests
+	return 0
+}
```

Comment:
I suggest this instead https://github.com/sirikid/void-packages/pull/5/files#diff-f84bf169ceb673e68584d5c25df39e898cf9f7cfe830f79c18ebe4f58f62f08eR26-R30

Reply:
Thanks for bisecting this, it saved a lot of time!